cmake/sasConfig.cmake.in

```diff
@@ -0,0 +1,43 @@
+# - Config file for the sas package
+# It defines the following variables
+#  sas_SCRIPT_DIR   - Python scripts
+#  sas_LIBRARIES    - libraries to link against
+#  sas_LIBRARY_DIR  - sas library dir
+#  sas_BINARY_DIR   - binary directory
+
+# Compute paths
+get_filename_component(sas_CMAKE_DIR "${CMAKE_CURRENT_LIST_FILE}" PATH)
+
+get_filename_component(sas_SCRIPT_DIR "${sas_CMAKE_DIR}/../scripts" ABSOLUTE)
+get_filename_component(sas_BINARY_DIR "${sas_CMAKE_DIR}/../bin" ABSOLUTE)
+get_filename_component(sas_LIBRARY_DIR "${sas_CMAKE_DIR}/../lib" ABSOLUTE)
+
+
+FIND_LIBRARY( sas_LIBRARIES NAMES libSas.so PATHS
+             ${sas_LIBRARY_DIR}
+             NO_DEFAULT_PATH
+        )
+
+INCLUDE( FindPackageHandleStandardArgs )
+FIND_PACKAGE_HANDLE_STANDARD_ARGS( sas DEFAULT_MSG sas_LIBRARY_DIR sas_SCRIPT_DIR sas_LIBRARIES )
+
+MACRO(enable_sas)
```

Comment:
... And add a comment about the arguments here. For an example, have a look [here](https://gitlab.cern.ch/jlingema/Gaudi/blob/master/cmake/GaudiProjectConfig.cmake#L2263)